data/categories.txt

```diff
@@ -536,7 +536,7 @@ nl:3^Tearoom|3brasserie|3café|3restaurant
 fi:2^Kahvila
 fr:3^Café|3restaurant|3resto
 de:3^Café|3Restaurant|Kaffee
-hu:3^Kávézó|3étterem|3élelmiszer
+hu:3^Kávézó|cukrászda|3étterem|3élelmiszer
```

Comment:
Are you sure? We do not include it in other languages, there is shop-confectionery for this.

Reply:
It is a constant question what Hungarian "cukrászda" is. A cukrászda in Hungary is actually a café: you enter, order your coffee from the waiter, chat with your friends etc. The only difference between a café and a cukrászda is that in a cukrászda you have (more and self-made) sweets (cakes, pies, biscuits and, in summertime, ice) available. However, the border between the two is vague. Therefore café and cukrászda may be together. A Hungarian cukrászda is not really a shop, it is more like a café or a restaurant (i.e. an amenity in OSM terms)

A shop=confectionery​ is a shop where you enter, buy your sweets and take them away, so no chairs and tables, no waiter, just like a real shop. This is a very rare species in Hungary and would be "édességbolt".